source/reference/minio-mc/mc-mirror.rst

```diff
@@ -191,44 +173,61 @@ Parameters
    
    Overwrites object(s) on the :mc-cmd:`~mc mirror TARGET`.
 
+   For example, object A exists on both the Source and the Target.
+   Object A on the source changes.
+   Running `mc mirror --overwrite` replaces the version on the target with the version from the Source.
+
 .. mc-cmd:: --preserve, a
    
 
-   Preserve file system attributes and bucket policy rules of the
-   :mc-cmd:`~mc mirror SOURCE` on the
-   :mc-cmd:`~mc mirror TARGET`.
+   Preserve file system attributes and bucket policy rules of the :mc-cmd:`~mc mirror SOURCE` on the :mc-cmd:`~mc mirror TARGET`.
 
 .. mc-cmd:: --region
    
 
-   Specify the ``string`` region when creating new bucket(s) on the
-   target. 
+   Specify the ``string`` region when creating new bucket(s) on the target. 
 
    Defaults to ``"us-east-1"``.
 
 .. mc-cmd:: --remove
    
 
-   Removes extraneous object(s) on the target. 
+   Removes object(s) on the Target that do not exist on the Source. 
+
+   For example, objects A, B, and C exist on Source.
+   Objects C, D, and E exist on Target.
+
+   When running `mc mirror --remove`, objects A and B copy to Target and objects D and E are removed from the Target.
+   C already exists on both, so does not change on either.
```

Comment:
Might clarify here that C must be identical on source and destination. If the C on the destination differs - actually I'm not sure what would happen there - might be worth testing. With --overwrite we would overwrite, but if we are just checking for filenames and not checksums, the behavior is less clear.